python/doc/pyplots/MetaModelValidation.py

```diff
@@ -0,0 +1,12 @@
+import openturns as ot
+from openturns.viewer import View
+
+dist = ot.Uniform(-3.0, 3.0)
+model = ot.SymbolicFunction(["x"], ["sin(x)"])
+metaModel = ot.SymbolicFunction(["x"], ["x - x^3 / 6.0 + x^5 / 120.0"])
```

Comment:
Would this one satisfy your criteria?
```python
metaModel = ot.SymbolicFunction(['x'], ['-1.287e-2 + 0.7525 * x + 9.343e-03 * x^2'])
```
I built it using `LinearLeastSquares` and a random sample of size 100.
Or, would you want me to insert the script that produces that particular surrogate, so that the user can see how to create a basic surrogate?
I wait for your answer before updating the content.

Reply:
In my opinion a minimalist script that produces the linear model would be optimal in order to show the most common usage of the proposed validation class.

Reply:
Ok. Fixed at a0b903fbd.